src/main/resources/metadata/tezos.carthagenet.conf

```diff
@@ -1,3 +1,4 @@
+edited
```

Comment:
What's this line? Is it an unwanted artifact?

I noticed it on the mainnet conf PR too, now that I checked again